### DISCUSSION TITLE
Welcome to TaqwaNet-MM-Core Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/TaqwaNet-MM#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Clawue884** [comment]
Perkenalkan nama saya cakar884(clawue)
Fokus utama proyek ini adalah menciptakan platform yang aman, terpercaya, dan sesuai Syariah, yang dapat diandalkan oleh pengguna dalam transaksi keuangan mereka.

Proyek yang sangat menarik,,semogasaja berjalan dan terealisasi

**Comment by azzdi1018** [comment]
hello
